app/models/role.rb

```diff
@@ -0,0 +1,30 @@
+# == Schema Information
+#
+# Table name: roles
+#
+#  id              :bigint           not null, primary key
+#  role            :integer
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#  organization_id :bigint           not null
+#  user_id         :bigint           not null
+#
+# Indexes
+#
+#  index_roles_on_organization_id  (organization_id)
+#  index_roles_on_user_id          (user_id)
+#
+class Role < ApplicationRecord
+  SUPERADMIN = 'superadmin'.freeze
+  ADMIN = 'admin'.freeze
+  MANAGER = 'manager'.freeze
+  USER = 'user'.freeze
+  USER_ROLES = [SUPERADMIN, ADMIN, MANAGER, USER].freeze
+
+  enum role: USER_ROLES
```

Comment:
так не гуд, краще так
```suggestion
  enum role: { superadmin: SUPERADMIN, admin: ADMIN ... }
```

Reply:
доречі, тобі не треба було міняти тип колонки, могла бути строка, а не число